.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
+
+on:
+  workflow_dispatch:
```

Comment:
I see the intent, but I think it should at least take a tag input? So that we can replicate the tag of the release.